modules/tinymce/src/plugins/table/test/ts/browser/InsertColumnTableSizeTest.ts

```diff
@@ -0,0 +1,200 @@
+import { Assertions, Chain, Guard, Log, NamedChain, TestLogs, UiFinder } from '@ephox/agar';
+import { Assert, UnitTest } from '@ephox/bedrock-client';
+import { McEditor, TinyDom } from '@ephox/mcagar';
+import { SelectorFind } from '@ephox/sugar';
+
+import Plugin from 'tinymce/plugins/table/Plugin';
+import SilverTheme from 'tinymce/themes/silver/Theme';
+import * as TableTestUtils from '../module/test/TableTestUtils';
+
+UnitTest.asynctest('browser.tinymce.plugins.table.InsertColumnTableSizeTest', (success, failure) => {
```

Comment:
It's hard to tell in GitHub, but is this file just renamed? If not, should this be using BDD syntax instead?

Reply:
Renamed, it had the name I wanted so I renamed it and used the name for my new tests. Git couldn't handle it

Reply:
Cool, thought that's all it was :+1: